server.js

```diff
@@ -1632,7 +1632,7 @@ cache(function(data, match, sendBadge, request) {
         sendBadge(format, badgeData);
       } else if (info === 'pyversions') {
         var versions = [];
-        var pattern = /^Programming Language \:\: Python \:\: (\d\.\d)$/;
+        var pattern = /^Programming Language \:\: Python \:\: (\d+\.*\d*)?$/;
```

Comment:
Why the `?` at the end? Isn't there always a version after `Python :: `?

Isn't this smaller regex sufficient?
`([\d\.]+)`